tests/datasets/test_xview.py

```diff
@@ -92,3 +92,94 @@ def test_plot(self, dataset: XView2) -> None:
         x['prediction'] = x['mask'][0].clone()
         dataset.plot(x)
         plt.close()
+
+
+class TestXView2DistShift:
+    @pytest.fixture(params=['train', 'test'])
+    def dataset(self, monkeypatch: MonkeyPatch, request: SubRequest) -> XView2DistShift:
+        monkeypatch.setattr(
```

Comment:
Might not even need to monkeypatch this if you remove `checksum=True`, I never bother with this